src/object/cli_obj.c

```diff
@@ -814,7 +814,7 @@ obj_shard_tgts_query(struct dc_object *obj, uint32_t map_ver, uint32_t shard,
 		D_GOTO(out, rc);
 
 	shard_tgt->st_rank	= obj_shard->do_target_rank;
-	shard_tgt->st_shard	= shard,
+	shard_tgt->st_shard	= obj_shard->do_shard;
```

Comment:
just curious any difference of using "shard" and "obj_shard->do_shard" at above L817 and L177? should be same?

Reply:
shard is the shard_idx in object layout.  but do_shard is the shard number.  see the example below.